### ISSUE TITLE
Flutter app not showing any output on dartpad

### ISSUE THREAD

**Issue by adnanbadshah** [dart-lang/dart-pad#1545]
I would like to see how it looks when my flutter app gets run on a device. On flutter's website, this seems to be possible. However, when I access dartpad.dartlang.org and try to run my app there, it shows nothing. Is there a way to make it work like that of flutter's website?

Im on pop os, based on debian linux. Accessing dartpad within firefox browser

**Comment by RedBrogdon** [comment]
Are you able to run the examples that appear in the dropdown at the top-right of dartpad.dev?

And are there any warnings or errors appearing in your browser's console?

**Comment by adnanbadshah** [comment]
It appears that when I changed my location on vpn, It worked fine. Otherwise it had some 404 error

**Comment by BRO3886** [comment]
Yes, isn't working for me as well

I'm getting a 400 for WS connection 

```bash
WebSocket connection to 'wss://gc.kis.v2.scr.kaspersky-labs.com/7D8B79A2-8974-4D7B-A76A-F4F29624C06BoVSE4_WV-C3WibZmeqfpGdUx1wPaPWDJSwWvEf7lBKdtO5JT0CXeqlNX24e2y9odVA7h8xDVzNwQ0pCsrPCcog/websocket?url=https%3A%2F%2Fdartpad.dartlang.org%2Fscripts%2Fframe_dark.html&nocache=1589887780254' 
failed: Error during WebSocket handshake: Unexpected response code: 400
GetWebSocket @ main.js?attr=O8AKXh6aJXfRQQM1-DKKnVrQtNiGMG5pXCkqEVtTLJ5gBRdUBrOyeEOzT2OI_FR6um9VJBN39nGweF215HJ1qTOAyT0iCBWHkEAOvvM9h7g:1462

```

and getting CORS errors

```bash
Access to XMLHttpRequest at 'https://dartpad.dartlang.org/scripts/assets/FontManifest.json' 
from origin 'null' has been blocked by CORS policy:Request header field x-requested-with is not allowed by Access-Control-Allow-Headers in preflight response.
```

**Comment by domesticmouse** [comment]
I’m curious why it is importing the resource from DartPad.dartlang.org instead of DartPad.dev.

The websocket connection to kaspersky-labs.com is total weirdness. I don’t think we use websockets in DartPad, so that’s probably a badly misbehaving browser plugin, at a guess.

**Comment by BRO3886** [comment]
The Kaspersky WS should be an issue on my device, I'm using it. My bad

**Comment by 0rbb** [comment]
Try run it in private window.

**Comment by BRO3886** [comment]
It works now after I removed Kaspersky. Works  fine. Thanks :)

**Comment by domesticmouse** [comment]
Closing this out for now. If you see a repeat of this, please open a new issue with steps to reproduce. Thanks!

**Comment by MilindGour9869** [comment]
When You don't have Kaspersky wait for around 1 minute , the code runs slow on web ...
